### PR TITLE
US6539 - Add loading spinner to initial app state

### PR DESCRIPTION
**Pull request by seancdavis**
Update the DDK to include loading indicator when the Angular application is loading - this story should govern the spinner icon only bc that is necessary for embed.

This implementation is simply the initial state of any application and is literally hardcoded right into index.html...

https://github.com/crdschurch/crds-embed/blob/development/src/index.html#L65-L101

---

This PR does not correspond with a change to crds-styles.

### CODE REVIEW

File: src/app/ui-components/loaders/icons/icons.component.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    transform:rotate(360deg);
+  }
+}
+.preloader &#123;
+  position: fixed;
+  width: 75px;
+  height: 75px;
+  background: transparent;
+  margin: 0 auto;
+  -webkit-animation: spin 700ms linear infinite;
```
Comment (seancdavis) [inline src/app/ui-components/loaders/icons/icons.component.html]: @zarahzachz: I pulled the example code directly from embed, but it seems like this should reference the `spinnerRotate` animation, right? (Or maybe `spin` and `spinnerRotate` animations should have the same name?)

**Comment by seancdavis** [comment]
@tcmacdonald @zarahzachz > Also, just remembered we have to write a SQL migration (probably not correct term) to make sure the jumbotron content block gets created in production.

File: src/app/ui-components/loaders/icons/icons.component.html (hunk 1 above)
Comment (tcmacdonald) [inline src/app/ui-components/loaders/icons/icons.component.html]: @seancdavis – Good call. Is spinnerRotate actually used anywhere within this implementation? If not, we should combine these declarations and delete the extraneous one.

File: src/app/ui-components/loaders/icons/icons.component.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          <path d="M50 98.842c27.614 0 50-22.127 50-49.42C100 22.125 77.614 0 50 0" stroke-opacity=".631" stroke="#3B6E8F"></path>
+        </g>
+      </g>
+    </svg>
+  </div>
+</div>
+<figure class="highlight">
+  <pre class="language-markup">
+&lt;style>
+@keyframes rotateInitLoadingSpinner &#123;
```
Comment (seancdavis) [inline src/app/ui-components/loaders/icons/icons.component.html]: @tcmacdonald: I changed the recommended name of the animation so as to not be too generic and conflict with another animation we may write in the future.

**Pull request merged**